api/controllers/AuthController.js

```diff
@@ -45,6 +46,9 @@ module.exports = {
         return res.redirect('/');
       });
     })(req, res);
+  },
 
+  modAuth: function(req, res) {
```

Comment:
This will callback to a page with an error message, which I couldn't figure out how to fix. The authentication still works, though.